plugins/pluginmanagers/lz-n.nix

```diff
@@ -208,8 +247,19 @@ lib.nixvim.plugins.mkNeovimPlugin {
 
   extraConfig = cfg: {
     globals.lz_n = lib.modules.mkAliasAndWrapDefsWithPriority id options.plugins.lz-n.settings;
-    plugins.lz-n.luaConfig.content = mkIf (cfg.plugins != [ ]) ''
-      require('lz.n').load( ${toLuaObject cfg.plugins})
-    '';
+    plugins.lz-n.luaConfig.content = mkMerge (
+      optional (cfg.plugins != [ ]) "require('lz.n').load(${toLuaObject cfg.plugins})"
+      ++ map (
+        {
+          plugin,
+          mode,
+          key,
+          action,
+          options,
+          ...
+        }:
+        "require('lz.n').keymap(${toLuaObject plugin}).set(${toLuaObject mode}, ${toLuaObject key}, ${toLuaObject action}, ${toLuaObject options})"
```

Comment:
Q: does `set` support any named args pattern?

Reply:
No, it doesn't